revocation/crl/bundle.go

```diff
@@ -0,0 +1,26 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package crl
+
+import (
+	"crypto/x509"
+)
+
+// Bundle is in memory representation of the Bundle tarball, including base CRL.
+//
+// TODO: consider adding DeltaCRL field in the future
+type Bundle struct {
+	// BaseCRL is the parsed base CRL
+	BaseCRL *x509.RevocationList
+}
```

Comment:
Since the `download` function of `HTTPFetcher` downloads both BaseCRL and DeltaCRL, why not storing all of them in the cache? We're just not using the `DeltaCRL` yet.
```suggestion
type Bundle struct {
	// BaseCRL is the parsed base CRL
	BaseCRL *x509.RevocationList
	
	// DeltaCRL is the parsed delta CRL
	DeltaCRL *x509.RevocationList
}
```

Reply:
download function only download `BaseCRL`. However, I added this field according to Shiwei's suggestion.